src/python_testing/matter_testing_support.py

```diff
@@ -1008,9 +1008,11 @@ def skip_all_remaining_steps(self, starting_step):
                 test is more deliberately identifying where test skips are starting from, making
```

Comment:
This comment is meant to be on line 1004. I think the parameter `starting_step` should be changed to `starting_step_number`

I find it harder to read when `starting_step` is overloaded as two different thing

Reply:
changed.